contracts/src/CrossChainApplications/examples/ERC20Bridge/tests/ERC20BridgeTests.t.sol

```diff
@@ -645,6 +645,11 @@ contract ERC20BridgeTest is Test {
         vm.expectCall(tokenContract, abi.encodeCall(token.decimals, ()));
     }
 
+    function _createMessageID(uint256 messageNonce) private pure returns (bytes32) {
```

Comment:
we should update to match new design

Reply:
Good call. I opted to remove `_createMessageID` all together in application-level tests in favor of just using a mock value. For for Teleporter tests, we need to calculate the expected values the contract will create, but for application tests, we mock out the value Teleporter returns either way. 👍